Sources/Prelude/KeychainClient/KeychainActor.swift

```diff
@@ -0,0 +1,156 @@
+import Foundation
+import KeychainAccess
+
+// MARK: - KeychainActor
+@_spi(KeychainInternal)
```

Comment:
is `@_spi` stable to be used? last time I did check it was an experimental feature.

Reply:
Why do we need it, for testing? Can't we solve it with testable?

Reply:
we dont wanna expose this to much, in fact it used to be completely hidden - thus untested - https://github.com/radixdlt/babylon-wallet-ios/blob/main/Sources/Prelude/KeychainClient/KeychainClient%2BLive.swift#L12

After the `Mono` PR has been merged, I can removed `@_spi`, since then I can use `testable import`, but it cannot be done now, since I cannot testable import another target, I think.